index.js

```diff
@@ -0,0 +1,126 @@
+require('dotenv').config();
+
+const express = require('express');
+const cors = require('cors');
+const bcrypt = require('bcryptjs');
+const jwt = require('jsonwebtoken');
+const db = require('./database/dbConfig.js');
+const server = express();
+
+server.use(express.json());
+server.use(cors());
+
+// ======== SERVER RUNNING ======== //
+server.get('/', (req, res) => {
+  res.send('<h1>Its Alive!<h1>');
+});
+
+
+// ========= CREATES USER ========= //
+server.post('/api/register', (req, res) => {
+  const credentials = req.body;
```

Comment:
Destructure the password, username, and department from the request body for database security purposes.